app/models/data_store_record.rb

```diff
@@ -0,0 +1,9 @@
+require "flex_commerce_api/api_base"
+
+module FlexCommerce
+  class DataStoreRecord < FlexCommerceApi::ApiBase
+    def self.table_name
+      'generic_data_store/record'
+    end
```

Comment:
I presume this is the path? The JSON API `type` may need changing too as this would default to `data_store_record` and I'm pretty sure the API returns as just `record`.